### ISSUE TITLE
Reddit crashes on link-click in iOS 9.3.3 (was working previously)

### ISSUE THREAD

**Issue by jacobseptember** [igiazlas/RedditSafariVC#1]
What it says in the title—when I click on a link in Reddit, the app crashes. This tweak was working for me previously.

Some quasi-scattershot testing reveals:

**WORKING**
Reddit 2.5.0.2057, RSVC 0.0.5
Reddit 2.6.0.00033, RSVC 0.0.5
Reddit 2.7.0.200180, RSVC 0.0.5

**NOT WORKING**
Reddit 2.5.0.2507, RSVC 0.0.8+
Reddit 2.6.1.200054, RSVC 0.0.8+
Reddit 2.7.0.200180, RSVC 0.0.8+

This is slightly strange, because I had thought I had v0.0.8 installed and working before I installed 0.0.9. Perhaps I'm misremembering.

Find on Pastebin [0.0.9 crashlog](https://pastebin.com/1ewJcnia), [0.0.9 syslog](https://pastebin.com/2uQuXNEz), [1.0.0 crashlog](https://pastebin.com/k52Sqa8y), [1.0.0 syslog](https://pastebin.com/XpNc21gi).

**Comment by igiazlas** [comment]
Hi, thanks for the feedback.

I made some changes in the 0.0.85 version to address a bug. This change made it incompatible with iOS 9. I’ll try to add support for iOS 9 as soon as possible. For the time being you can stay on 0.0.8 (you can get the deb from here <https://github.com/igiazlas/RedditSafariVC/releases>)

**Comment by jacobseptember** [comment]
Thanks—

0.0.8 also crashes Reddit, which is strange, because I really did think I was using it before. I'll go back to 0.0.5 for the time being.

**Comment by ghost** [comment]
Same issue for me. Please fix ASAP.

No offense but excluding everyone on iOS 9 for fixing a single bug seems kinda.. off..

**Comment by igiazlas** [comment]
You’re right. I didn’t realize I was breaking iOS 9 support and also I don’t have access to an iOS 9 device to test. I’m attaching two builds. Can you check if they are working?

Thank you

**Comment by ghost** [comment]
I'd be happy to help, yes ;)

**Comment by ghost** [comment]
With version 1.0.1, the issue still exists (iOS 9.3.3, iPhone 6).

Although I'm not sure the reddit app is actually crashing.. when i tap on a link it just exits the app to the homescreen as if i had pressed the home button. But I'm no dev, so no clue..

**Comment by jacobseptember** [comment]
Agreed, 1.0.1 crashes Reddit still. [Crashlog.](http://pastebin.com/3Gd2PZim)

**Comment by igiazlas** [comment]
This update was for an unrelated bug.

Did you try the debs I sent you?

**Comment by jacobseptember** [comment]
Any debs included in your email back to the thread didn't make it through. Can you attach directly to the github issue?

**Comment by igiazlas** [comment]
I didn't even realized you opened an issue. I thought we were talking through good ol' email 😛.

There you go [debs.zip](https://github.com/igiazlas/RedditSafariVC/files/917916/debs.zip)

**Comment by jacobseptember** [comment]
Both 1.0.0-3+debug and 1.0.0-4+debug work perfectly for me! Thanks

**Comment by igiazlas** [comment]
The problem with those is that the color of the safariVC doesn't change with the theme
